src/main/java/thaumic/tinkerer/common/block/fire/BlockFireEarth.java

```diff
@@ -55,33 +57,39 @@ public ThaumicTinkererRecipe getRecipeItem() {
                 new AspectList().add(Aspect.FIRE, 5).add(Aspect.MAGIC, 5).add(Aspect.EARTH, 5));
     }
 
-    @Override
-    public HashMap<BlockTuple, BlockTuple> getBlockTransformation() {
-        HashMap<BlockTuple, BlockTuple> result = new HashMap<BlockTuple, BlockTuple>();
-        result.put(new BlockTuple(Blocks.sand), new BlockTuple(Blocks.dirt));
-        result.put(new BlockTuple(Blocks.gravel), new BlockTuple(Blocks.clay));
-        result.put(new BlockTuple(Blocks.nether_brick), new BlockTuple(Blocks.planks));
-        result.put(new BlockTuple(Blocks.nether_brick_fence), new BlockTuple(Blocks.fence));
-        result.put(new BlockTuple(Blocks.nether_brick_stairs), new BlockTuple(Blocks.oak_stairs));
-        result.put(new BlockTuple(Blocks.cactus), new BlockTuple(Blocks.log));
-        result.put(new BlockTuple(Blocks.snow_layer), new BlockTuple(Blocks.tallgrass));
-        result.put(new BlockTuple(Blocks.stone), new BlockTuple(Blocks.dirt));
-        result.put(new BlockTuple(Blocks.mob_spawner), new BlockTuple(Blocks.iron_block));
-        result.put(new BlockTuple(Blocks.log), new BlockTuple(Blocks.dirt));
+    private static void initBlockTransformation() {
+        blockTransformation = new HashMap<>();
+        blockTransformation.put(Blocks.sand, Blocks.dirt);
+        blockTransformation.put(Blocks.gravel, Blocks.clay);
+        blockTransformation.put(Blocks.nether_brick, Blocks.planks);
+        blockTransformation.put(Blocks.nether_brick_fence, Blocks.fence);
+        blockTransformation.put(Blocks.nether_brick_stairs, Blocks.oak_stairs);
+        blockTransformation.put(Blocks.cactus, Blocks.log);
+        blockTransformation.put(Blocks.snow_layer, Blocks.tallgrass);
+        blockTransformation.put(Blocks.stone, Blocks.dirt);
+        blockTransformation.put(Blocks.mob_spawner, Blocks.iron_block);
```

Comment:
We might need to put the terra one on the tinker block as well, these are really strange